core/postsolve.gms

```diff
@@ -12,36 +12,6 @@
 p_taxCO2eq_iteration(iteration,ttot,regi) = pm_taxCO2eq(ttot,regi);
 pm_taxemiMkt_iteration(iteration,ttot,regi,emiMkt) = pm_taxemiMkt(ttot,regi,emiMkt);
 
-*RP* added the historic 2010/2015 CO2 prices 
```

Comment:
I'm absolutely not sure, but isn't the problem that now in cm_emiscen != 9, we don't have Price|Carbon correctly reported? I would think we have to put here:
```
if (cm_emiscen eq 6),
$ifthen.neg %optimization% == 'negishi'     
 pm_taxCO2eqSum(ttot, regi) = abs(q_co2eq.m(ttot,regi)) / pm_ts(ttot) / (pm_pvp(ttot,"good") + sm_eps);
$else.neg
 pm_taxCO2eqSum(ttot, regi) = abs(q_co2eq.m(ttot,regi)) / (abs(qm_budget.m(ttot,regi) )+ sm_eps); 
$endif.neg
```
etc., not sure about the other cases. No idea why we have the `abs` stuff here, that looks rather suspicious to me…

Reply:
well, pm_taxCO2eqSum is calculated for all emiscen cases. I think, the only problematic case is cm_emiscen eq 6. I will check, if this is still needed

Reply:
Is it? Where? Because in the tax module, it seems it is simply an aggregation of `pm_taxCO2eq + pm_taxCO2eqRegi + pm_taxCO2eqSCC`, coming from the 45 module. But all these modules only work with emiscen = 9 (well, and 10 of course). So I guess with emiscen = 6, you will end up with `pm_taxCO2eqSum` = 0, if you don't overwrite it here.

Reply:
I agree, I hoped that we could delete (all) other emiscen scenarios. But seems we need to keep them, so added calculation of pm_taxCO2eqSum for more emiscen cases